ymcatwincities/openy_activity_finder
# PRODDEV-155 - zero result page.

Steps for review:
- [x] Solution A. go through all steps in the ticket description
- [x] Solution B. 
- Choose the evening of Sunday
- if shoould be displayed 'Clear filter' button https://i.imgur.com/LSzWXix.png -> click on it should clear all filters
- check some other filter f.e. age -> two buttons, that will clear all filters except specified in button name, should be displayed https://i.imgur.com/2q76M9a.png
